platform-includes/getting-started-use/javascript.nestjs.mdx

```diff
@@ -1,24 +1,57 @@
-```javascript {filename: main.ts} {17}
+```javascript {filename: main.ts}
 // Import this first!
 import './instrument';
 
 // Now import other modules
-import * as Sentry from "@sentry/nestjs";
-import {
-  BaseExceptionFilter,
-  HttpAdapterHost,
-  NestFactory
-} from '@nestjs/core';
+import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  const { httpAdapter } = app.get(HttpAdapterHost);
-
-  Sentry.setupNestErrorHandler(app, new BaseExceptionFilter(httpAdapter));
-
   await app.listen(3000);
 }
 
 bootstrap();
 ```
+
+Then you can add the SentryModule as a root module:
+
+<Note>The SentryModule needs to be registered before any other module that should be instrumented by Sentry.</Note>
+
+```javascript {filename: app.module.ts} {2, 8}
+import { Module } from '@nestjs/common';
+import { SentryModule } from '@sentry/nestjs/setup';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+
+@Module({
+  imports: [
+      SentryModule.forRoot(),
+      // ...other modules
+  ],
+  controllers: [AppController],
+  providers: [AppService],
+})
+export class AppModule {}
+```
+
+By default, exceptions with status code 4xx are not sent to Sentry. If you still want to capture these exceptions, you can do that manually with `Sentry.captureException()`:
```

Comment:
```suggestion
By default, exceptions that are caught by your own exception filters are not reported to Sentry. If you still want to capture these exceptions, you can do so manually with `Sentry.captureException()`:
```

Reply:
@lforst I don't think that is accurate, because if you rethrow a 500 in an exception filter then that will still be reported. Only if you catch an exception and then throw a different exception the original exception won't be reported. So left the 4xx text but simplified the example, should be fine for users to get the gist